inject.go

```diff
@@ -58,9 +72,13 @@ func Inject(target interface{}) Option {
 	// 		Baz io.Writer
 	// 	}
 	//
-	// The generated function has the shape,
+	// The generated struct has the shape,
 	//
-	// 	func(io.Reader, io.Writer)
+	// 	struct {
+	// 		fx.In
```

Comment:
Let's be consistent everywhere and add a space after `fx.In` and `fx.Out` embeds.